README.md

```diff
@@ -42,14 +42,6 @@ module "dynamodb_table" {
 The [change log](https://github.com/terraform-aws-modules/terraform-aws-dynamodb-table/tree/master/CHANGELOG.md) captures all important release notes.
 
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-
-## Requirements
```

Comment:
What's the deal here? My version of `terraform-docs` adds this?

Reply:
Good question. I just ran `pre-commit run -a` locally to fix some formatting and it removed these.

Reply:
I don't see what has been the reason for such behavior on GitHub actions in first place. Run 30 and 31 have no difference, but starting from run 31 GitHub actions failing lint. Weird.

https://github.com/terraform-aws-modules/terraform-aws-dynamodb-table/runs/551550821?check_suite_focus=true